docs/how_it_works/devkit/pce.mdx

```diff
@@ -0,0 +1,73 @@
+# How Privacy Computation Engine Works
+
+## Configuring your Privacy Computation Engine
+
+You must first configure the Privacy Computation Engine in order to make the most of it.
+It needs to know about:
+- General information about your Organization and its' data processing policies (Equivalent to a RoPA - Record of Processing Activities)
+- Mappings of your data structures to [PRIV](https://github.com/blindnet-io/product-management/blob/main/refs/schemas/priv/RFC-PRIV.md) Data Categories
+- Few choices you have to make about when to authenticate the user (before or after they make a Privacy Request), and when to deliver automatic responses and when to wait for validation
+
+To Configure your Privacy Computation Engine prepare the following information:
```

Comment:
Also `data consumer categories` according to [ROPA](https://transcend.io/record-of-processing-activities-ropa/)

Reply:
True. Added.